frame/bags-list/src/list/mod.rs

```diff
@@ -384,6 +384,77 @@ impl<T: Config> List<T> {
 		})
 	}
 
+	/// Put `heavier_id` to the position directly in front of `lighter_id`. Both ids must be in
+	/// same bag  and the `weight_of` `lighter_id` must be less than that of `heavier_id`
```

Comment:
I won't complain about inline docs (`//`), but these sentences that go in the public docs should be punctuated. 

```suggestion
	/// same bag  and the `weight_of` `lighter_id` must be less than that of `heavier_id`.
```

That being said, I would appreciate if the inline docs follow the same :p

Reply:
45f4a73e8c15ef9922bd9a961e554303918525ca